### DISCUSSION TITLE
SEB

### DISCUSSION THREAD

**Discussion by Rasulbek722** [SafeExamBrowser/seb-mac#394] [Q&A]
I downloaded SEB for my macbook, but when I open it shows Demo testing exam portal.How can I get rid of it

**Comment by danschlet** [comment]
It's not possible that it just shows the demo testing portal on its own, you must have clicked on some links which configured SEB to show the portal.

See here how to reset SEB client settings: 
https://github.com/SafeExamBrowser/seb-mac/issues/393
